docs/src/pages/Integrations.vue

```diff
@@ -108,7 +108,7 @@ export default defineComponent({
 
   &--bordered {
     border-radius: 8px;
-    border: solid 1px rgba(0, 0, 0, 0.12);
+    border: solid 1px rgba($black, 0.12);
```

Comment:
Zeplin is wrong here, seems like the correct color to apply here is `border: solid 1px rgba($brand-secondary, 0.54);`

It's probably due to multiple layers one over the other in how the component has been created in Sketch by Nick